packages/ui-app/src/Notification.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BareProps } from './types';
+
+import React from 'react';
+
+import Message from 'semantic-ui-react/dist/commonjs/collections/Message/index';
+import classes from './util/classes';
```

Comment:
Extra blank line before this line, i.e. split external and per-package imports into seperate sections.